app/views/layouts/application.html.erb

```diff
@@ -15,16 +15,16 @@
     <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-167x167.png'), rel: 'apple-touch-icon', type: 'image/png', size: '167x167' %>
     <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
     <%= stylesheet_pack_tag 'application', media: 'all' %>
-    <%= javascript_pack_tag 'application', defer: true %>
+    <%= javascript_pack_tag 'application', defer: true unless params[:nojs] === "nojs" %>
```

Comment:
Why?

Reply:
So that we can test the no-JS functionality in Cypress - it doesn't have a way of running tests without JavaScript.

I thought about removing the conditional on production, but I really see any reason to - it might help with debugging an issue one day, and if a user does somehow guess it the site will still work anyway.

Reply:
That sounds reasonable. I think we should avoid using this during any manual testing though. I don't trust it completely

Reply:
For sure. We also lose it during page navigation.